### DISCUSSION TITLE
Theme widget category for dragging files into containing folders

### DISCUSSION THREAD

**Discussion by scott092707** [mate-desktop/caja#1782] [General]
I just installed a gtk theme ("Ambiance-RW") and then modified it so that the scrollbars are more visible on my 49" UHD TV-as-monitor.
However, the theme does not have what the previous one had: when I select files and drag them into a folder on the location bar,
the folder-button's border became green, so that I knew it hit it right.  Now, it does not change at all...
I don't know themes much at all, and have no idea what type of widget (in gtk-widgets.css, I assume...) I should look for, and what properties.
Dragging folders into the button is not quite "hovering", and not quite "selected", ...

Could anyone tell me what I should be looking for?

I guess, since the button doesn't change when I drag stuff into it (this still works - it just does not indicate it),
it is possible that that option is not in the file at all, and will have to be added...

Help, anyone ... ?

**Comment by lukefromdc** [comment]
Compare whatever you are having problems with between your theme and any theme that works, such as the MATE themes. Then open GtkInspector (you may have to enable debug for gtk in dconf-editor) and find the offending element.

Sorry, I am not an effective teacher and cannot provide a tutorial in how to do this

**Reply by scott092707** [reply]
This is the only gtk theme I have downloaded - the only theme already here was Raleigh, and “Default” and all I get from looking at files there, is:

```
#
# This theme is the default theme if no other theme is selected.
#
```
and

```
/*
 * Default keybinding set. Empty because it is implemented inline in the code.
 */
```

i.e. there is no other theme to which I can compare.
I have to assume that the behaviour that existed prior to downloading/installing "Ambiance-RW", is whatever Caja defaults to when it cannot find a theme to access - something built-in to Caja.

BTW: Debian Testing on Sway 1.9-1+b1; caja 1.26.3-1+b1

**Comment by lukefromdc** [comment]
Try installing the mate-theme package for testing, you can remove it after you have your (non-MATE supporting) theme fixed. 
Be sure to back up all modded theme files and/or copy the theme to a new name, otherwise one package update will overwrite
all your changes.

**Reply by scott092707** [reply]
I installed Debian's mate-themes package, and using 'env GTK_THEME=' with each of the themes that were gtk-3.x-capable, found that 'Yaru*' themes had the drag-target highlighting I remembered from before.  gtk's forum suggested that what I wanted would be referenced by ':drop(active)'.  I ended up grep'ing all  ':drop(active)' lines, and copied in all lines that also referred to a color (GIMP showed that it was a pale green) into my theme, changing the color to something more intense, and adding a doubled border of the same color.

Thank you for your suggestion.

[I did enable/use GtkInspector, and while I found it interesting, I found myself more confused than enlightened... ]
